src/main/webapp/i18n/en/quizExercise.json

```diff
@@ -176,7 +176,11 @@
                 "questionTitleLength": "Question {{index}}: The question is too long. Please shorten it to fewer than 250 characters.",
                 "quizTitleLength": "Quiz: The title is too long. Please shorten it to fewer than 250 characters.",
                 "explanationIsMissing": "Question {{index}}: Not every answer option contains an explanation. Please add an explanation!",
-                "questionHasInvalidFlaggedElements": "<strong>Question {{index}} </strong>: This question has been previously marked as invalid or contains invalid elements."
+                "questionHasInvalidFlaggedElements": "<strong>Question {{index}} </strong>: This question has been previously marked as invalid or contains invalid elements.",
+                "questionExplanationLength": "Question {{index}}: The provided explanation is too long. Please shorten it to fewer than 255 characters.",
```

Comment:
did you forget to add german translations for these newly created strings?

Reply:
Oh thanks for the hint, I indeed forgot the german translation. This is fixed now.
To your more general feedback: Yes, I can understand your personal preference and I am open for a discussion on this. I implemented it this way because the a quiz title that exceeds a length of 250 characters is already handled like this and I wanted the exception handling to stay uniformly.